fishtest/fishtest/templates/nns.mak

```diff
@@ -45,6 +50,15 @@ The recommended net for a given Stockfish executable can be found as the default
    <td style="text-align:right">${nn.get('downloads', 0)}</td>
   </tr>
   %endif
+  %endif
  %endfor
  </tbody>
 </table>
+<p>
```

Comment:
This seems to work on DEV:
```
<p>
 %if prev_page:
  <a href="nns?page=${prev_page}">&lt Later networks</a>
 %endif
 %if prev_page and next_page:
  &#8209;
 %endif
 %if next_page:
  <a href="nns?page=${next_page}">Earlier networks &gt</a>
 %endif
</p>
```

Reply:
@ppigazzini Looks much better! Thanks, will add that.

Reply:
Please double check my syntax :)

Reply:
I spotted the missing `;` but for some reason `&#8209;` shows also red in my FF page source view?

Reply:
Perhaps `<span>&#8209;</span>` ?
https://www.toptal.com/designers/htmlarrows/punctuation/non-breaking-hyphen/

Reply:
DEV updated with `<span>` tag.

Reply:
`<span>`looks good

Reply:
maybe 'newer' and 'older' are better than 'earlier' and 'later' ...

Reply:
@vondele done.

![image](https://user-images.githubusercontent.com/15718418/96519343-118b2900-126d-11eb-9cb8-6cf62d97dd35.png)